homeassistant/components/husqvarna_automower/config_flow.py

```diff
@@ -0,0 +1,57 @@
+"""Config flow to add the integration via the UI."""
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import data_entry_flow
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import config_entry_oauth2_flow
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class HusqvarnaConfigFlowHandler(
+    config_entry_oauth2_flow.AbstractOAuth2FlowHandler,
+    data_entry_flow.FlowHandler,
+    domain=DOMAIN,
+):
+    """Handle a config flow."""
+
+    DOMAIN = DOMAIN
+    VERSION = 1
+
+    async def async_oauth_create_entry(self, data: dict) -> FlowResult:
+        """Create an entry for the flow."""
+        return await self.async_step_finish(DOMAIN, data)
+
+    async def async_step_finish(self, unique_id, data) -> FlowResult:
+        """Complete the config entries."""
+        await self.async_set_unique_id(unique_id)
+
```

Comment:
```suggestion
            self._abort_if_unique_id_configured()
```
This will raise an issue if the unique_id is already set. This way you can't login twice with the same account.